shapes.go

```diff
@@ -187,6 +187,39 @@ func (p *Proc) Triangle(x1, y1, x2, y2, x3, y3 float64) {
 	)
 }
 
+func (p *Proc) Bezier(x1, y1, x2, y2, x3, y3, x4, y4 float64) {
+	if !p.doStroke() {
+		return
+	}
+
+	var (
+		sp  = p.pt(x1, y1)
+		cp0 = p.pt(x2, y2).Sub(sp)
+		cp1 = p.pt(x3, y3).Sub(sp)
+		ep  = p.pt(x4, y4).Sub(sp)
+	)
+
+	path := func(o *op.Ops) clip.PathSpec {
```

Comment:
I don't think wrapping the path specification inside a closure is needed there.
it was needed in the case of the vertices API because one needed to "replay" the path for the outline and/or for the strokes.

but here, if I am not mistaken, one just has to build the path once and commit it as a stroked path.

so:
```go
var path clip.Path
...

clip.Stroke{
    Path: path.End(),
    ...
```

Reply:
fixed